easybuild/easyconfigs/c/CUDA/CUDA-7.5.18-iccifort-2015.3.187-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 Cyprus Institute / CaSToRC, University of Luxembourg / LCSB, Ghent University
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis.georgatos@uni.lu>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-99.html
+##
+
+#easyblock = 'ConfigureMake'
+
+name = 'CUDA'
+version = '7.5.18'
+
+homepage = 'https://developer.nvidia.com/cuda-toolkit'
+description = """CUDA (formerly Compute Unified Device Architecture) is a parallel
+ computing platform and programming model created by NVIDIA and implemented by the
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = {'name': 'iccifort', 'version': '2015.3.187-GCC-bare-4.9.3'}
+
+# eg. http://developer.download.nvidia.com/compute/cuda/6_5/rel/installers/cuda_6.5.14_linux_64.run
+#source_urls = ['http://developer.download.nvidia.com/compute/cuda/7_5/rel/installers/']
+source_urls = ['http://developer.download.nvidia.com/compute/cuda/7.5/Prod/local_installers/']
+
+sources = ['%(namelower)s_%(version)s_linux.run']
+checksums = [('md5', '4b3bcecf0dfc35928a0898793cf3e4c6')]
+
+installopts = '-override compiler'
```

Comment:
can you add a comment explaining why this is needed? the other CUDA easyconfigs don't have it